### PR TITLE
Readme updates + updates needed to run app

### PR DESCRIPTION
**Pull request by whatSocks**
* clarified that migrations will probably needed
* emphasized filling in certain env variables
* added link to discord (but split up the URL to thwart bots)
* updated dependencies to be able to run frontend

### CODE REVIEW

**Comment by whatSocks** [comment]
note that it would be nice to not have the ordered list on the readme

File: README.md [hunk 1]
```diff
@@ -62,24 +73,34 @@ RxC Voice - http://localhost:3000
 
 ## Creating users and accessing the site for testing
 
-1. Create a superuser to access the admin site
+1. Run migrations
+
+```
+docker exec -it rxc-voice_api_1 python manage.py migrate
```
Comment (alexrandaccio) [inline README.md:79]: migrations are run automatically on startup-- see rxc-voice/scripts/deploy/api_deploy_dev.sh

**Comment by alexrandaccio** [comment]
thanks this is helpful. will test soon to see if the package versions you chose work for me

File: README.md (hunk 1 above)
Comment (whatSocks) [reply README.md:79]: I was prompted to do it manually and couldn't run the app until i did so

**Comment by whatSocks** [comment]
> thanks this is helpful. will test soon to see if the package versions you chose work for me

for some context, I couldn't get it to run with the listed dependencies. 

To fix this, I did `npx depcheck` to check which weren't being used, then upgraded dependencies / added back dependencies until it worked.

**Comment by alexrandaccio** [comment]
Hmm, the master branch works for me and not for you, and this branch works for you and not for me. Isn't package management fun? We'll just have to put our heads together and figure this out in our meeting

**Comment by alexrandaccio** [comment]
@whatSocks why'd you decide to close this? I was planning on merging it once I reconciled the dependency problems we've been having

**Comment by whatSocks** [comment]
oops

**Comment by alexrandaccio** [comment]
merged in #71

**Pull request closed without merging**